packages/webview_flutter/webview_flutter_wkwebview/darwin/Classes/FLTWebViewFlutterPlugin.m

```diff
@@ -106,10 +125,12 @@ + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
   SetUpFWFNSUrlHostApi(registrar.messenger,
                        [[FWFURLHostApiImpl alloc] initWithBinaryMessenger:registrar.messenger
                                                           instanceManager:instanceManager]);
+#if TARGET_OS_IOS
   SetUpFWFUIScrollViewDelegateHostApi(
```

Comment:
is there a macos version of scroll view delegate?

Reply:
AFAICT the only way to implement scroll functionality on the macOS version of WKWebView without mucking around with internal view details is JS.